src/containers/LoginRegister/index.jsx

```diff
@@ -26,22 +26,22 @@ const LoginRegisterPage = (props) => {
       setError(e.message);
     }
   };
-  const passwordReset = async (email) => {
+  const resetPassword = async (email) => {
     setError("");
     setMessage(<></>);
     try {
       setLoading(true);
-      await auth.sendPasswordResetEmail(email);
+      await auth.sendResetPasswordEmail(email);
       setLoading(false);
       setMessage(
         <Alert
           style={{ marginBottom: 10 }}
           showIcon
           message={
             <div>
-              {t("loginRegister.anEmailsent")}
+              {t("loginRegister.anEmailSent")}
```

Comment:
"anEmailSent" key is written differently in the translation files, you may fix it.